Makefile

```diff
@@ -199,6 +151,7 @@ lint-js:
 	@echo ""
 
 
+# https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html
 .PHONY: bootstrap \
```

Comment:
Just remove all these phony targets except `build`. They're not realistic, and the effort to maintain this just isn't worth it. See https://github.com/getsentry/sentry/pull/24612#pullrequestreview-618189325.

Reply:
Why `build` is worth keeping around? Is there something special about it? Or do we need at least a list of 1?

Reply:
Guessing here, but out of all of these targets, `build` has the highest chance of being an actual file in the fs.

Reply:
Billy's right.